### ISSUE TITLE
Dynamically generated test names reported as duplicates

### ISSUE THREAD

**Issue by Mahoney** [kotest/kotest#5491]
The kotest plugin puts red squiggles all over this, but it runs fine:
```kotlin
import io.kotest.core.spec.style.StringSpec
import io.kotest.matchers.ints.shouldBeGreaterThan
import io.kotest.matchers.shouldBe

class IntegerSpec : StringSpec({

  listOf(
    1,
    2
  ).forEach { number ->
    "$number should be positive" {
      number shouldBeGreaterThan 0
    }
  }

  listOf(
    2,
    4
  ).forEach { number ->
    "$number should be even" {
      number % 2 shouldBe 0
    }
  }
})
```

**Comment by james-current** [comment]
Any ETA on when the fix will be pushed out? Every unit test in our codebase has this issue (I'm in process on upgrading our codebase from Kotlintest to Kotest)

**Comment by sksamuel** [comment]
It's fixed in the 1.1.0 plugin, but that needs 5/6 days more testing before
I can submit it to the plugin store.

**Comment by sksamuel** [comment]
Please try 1.1.9 that's now released.
